flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/files/StaticFileServerHandler.java

```diff
@@ -177,7 +187,11 @@ public void channelRead0(ChannelHandlerContext ctx, Routed routed) throws Except
 
 		HttpResponse response = new DefaultHttpResponse(HTTP_1_1, OK);
 		setContentTypeHeader(response, file);
-		setDateAndCacheHeaders(response, file);
+		// TODO This is counter-intuitive. Will be changed later when a dynamic file server is added
```

Comment:
This is to prevent the browser from caching the log or out files since they're highly dynamic. The refresh button will have no effect whatsoever otherwise.